stories/Display/ModalStory.tsx

```diff
@@ -8,18 +8,40 @@ import Button from '../../src/General/Button';
 class ModalStory extends React.Component {
   state = {
     visible: false,
+    visibleCenteredModal: false,
+    visibleCenteredModalWithMoreText: false,
   };
 
   handleOpen = () => {
-    this.setState({ visible: true });
+    this.setState({ ...this.state, visible: true });
```

Comment:
Oh for setState, I think we don't need to destructure other states when updating one root property.

Reply:
It's not required. I have a habit of destructuring this haha.